paymentintent.go

```diff
@@ -654,7 +668,7 @@ type PaymentIntentProcessingCard struct{}
 // If present, this property tells you about the processing state of the payment.
 type PaymentIntentProcessing struct {
 	Card *PaymentIntentProcessingCard `json:"card"`
-	Type string                       `json:"type"`
+	Type PaymentIntentProcessingType  `json:"type"`
```

Comment:
This change is fine since we haven't actually released https://github.com/stripe/stripe-go/pull/1393 yet.